python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -715,9 +715,11 @@ def getMetadataCircle(coord, radius, filterName, epoch=None):
         md.add('RA', coord.getRa().asDegrees(), 'field center in degrees')
         md.add('DEC', coord.getDec().asDegrees(), 'field center in degrees')
         md.add('RADIUS', radius.asDegrees(), 'field radius in degrees, minimum')
-        md.add('SMATCHV', 1, 'SourceMatchVector version number')
+        # Version 2 uses EQUINOX for the TAI Julian Epoch year.
+        md.add('SMATCHV', 2, 'SourceMatchVector version number')
         md.add('FILTER', filterName, 'filter name for photometric data')
-        md.add('EPOCH', "NONE" if epoch is None else epoch.mjd, 'Epoch (TAI MJD) for catalog')
+        md.add('EQUINOX', "NONE" if epoch is None else epoch.jyear,
```

Comment:
I think the standard now requires us to use `JEPOCH` here or `MJD-OBS`

Reply:
I've updated this to JEPOCH, cleaned up some docs, and added a trivial test of `getMetadataCircle`. Please have another look.